React/Modules/RCTTiming.m

```diff
@@ -151,7 +162,7 @@ - (void)stopTimers
 
 - (void)startTimers
 {
-  if (!_bridge || _timers.count == 0) {
+  if (!_bridge || (!_sendIdleEvents && _timers.count == 0)) {
```

Comment:
Might want to define a little helper method that returns `!_sendIdleEvents && _timers.count == 0`